third_party/move/move-vm/runtime/src/storage/dummy.rs

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) The Move Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    loader::{Module, Script},
+    storage::{module_storage::ModuleStorage, script_storage::ScriptStorage, verifier::Verifier},
+};
+use move_binary_format::{
+    errors::{PartialVMError, PartialVMResult},
+    file_format::CompiledScript,
+    CompiledModule,
+};
+use move_core_types::{
+    account_address::AccountAddress, identifier::IdentStr, metadata::Metadata,
+    vm_status::StatusCode,
+};
+use std::sync::Arc;
+
+macro_rules! not_implemented {
+    () => {
+        Err(PartialVMError::new(StatusCode::FEATURE_UNDER_GATING)
+            .with_message("New loader and code cache are not yet implemented".to_string()))
+    };
+}
+
+/// Dummy implementation of code storage, to be removed in the future. Used as a placeholder
+/// so that existing APIs can work
+pub struct DummyStorage;
```

Comment:
nit: DummyCodeStorage more descriptive maybe?

Reply:
done

Reply:
Oh, missed that, fixed now, thanks for flagging in the comment.